tests/driver_ph_oem/main.c

```diff
@@ -25,7 +25,7 @@
 #include "ph_oem_params.h"
 #include "ph_oem_regs.h"
 
-#define SLEEP_SEC                   (5)
+#define SLEEP_USEC_SEC                   (5)
```

Comment:
There's no need to change this one, no ?

Reply:
Oops, fixed